README.md

```diff
@@ -100,6 +100,11 @@ if no file is found). It is mutually exclusive with `:load-fn!` and
 will be ignored in case both are present
 
 * `:src-paths`  a vector of paths containing source files
+
+* `:cache` - a map containing two optional values: the first, `:path`, indicates
+the path of the cached files. The second, `:search-src-paths?`, indicates
```

Comment:
* yes, it's a boolean, hence the `?`
* yes, it works if you want both, in the tests I checked for many variants
.. let me just add in the docs the order of searching